src/components/UserSelector/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { DropdownItem } from './DropdownItem';
+import { User } from '../../../types/User';
+
+interface DropdownMenuProps {
+  users: User[];
+  setIsActive: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+interface UserWithKey extends User {
+  key: string;
+}
```

Comment:
Defining a separate interface for UserWithKey is unnecessary since you are only adding a key for rendering purposes. Consider using the key directly in the map function without modifying the user object.